### ISSUE TITLE
Is enum supported?

### ISSUE THREAD

**Issue by cihad** [remoteoss/json-schema-form#96]
Why is the `enum` not working?

```json
{
  "type": "object",
  "properties": {
    "city": {
      "title": "City",
      "enum": ["Foo", "Bar"],
      "x-jsf-presentation": {
        "inputType": "select"
      },
      "type": "string"
    }
  }
}
```
but it is ok:

```json
{
  "type": "object",
  "properties": {
    "city": {
      "title": "City",
      "oneOf": [
        { "const": "Foo", "title": "Foo" },
        { "const": "Bar", "title": "Bar" }
      ],
      "x-jsf-presentation": {
        "inputType": "select"
      },
      "type": "string"
    }
  }
}
```

**Comment by ollyd** [comment]
Hi @cihad! In your example `enum` won't work as it would need a corresponding `options` property within the `x-jsf-presentation` property. However, whilst this works our recommended implementation is using `oneOf` as per your second example. 

If you need to use `enum` in the way you've suggested, whereby the label would also represent the value, contributions to the library are always very welcome!

**Comment by cihad** [comment]
[Enum](https://www.learnjsonschema.com/2020-12/validation/enum/) will always come up. It would be good to support it. Maybe `enumLabels` can be added to `x-jsf-presentation`.

**Comment by ollyd** [comment]
> [Enum](https://www.learnjsonschema.com/2020-12/validation/enum/) will always come up. It would be good to support it. Maybe `enumLabels` can be added to `x-jsf-presentation`.

Hi @cihad agree it would be good to support it. We would suggest `enumTitles` rather than `enumLabels` as `title` is more in line with the json schema spec. If you would like to take a stab at opening a PR, it would be appreciated.

**Comment by sandrina-p** [comment]
@cihad one more thing we remembered:

Imagine this `enum: ["1", 1, true, "true"]` Using `enumTitles` an object, we cannot match the title for each key because they are always strings.  So the enumTitles needs to also be an array.
In your example, it would be `enumTitles: ["Identification", "P45 Document"]`

Does this make sense to you?

**Comment by cihad** [comment]
Yes, it would be better to use `enumTitles` as an array instead of an object.

**Comment by sandrina-p** [comment]
Hi @cihad, we added support to `enum` in v1 at https://github.com/remoteoss/json-schema-form/pull/181. It doesn't include the `enumTitles`, but it works for simpler cases as yours `"enum": ["Foo", "Bar"]` :)
